R/project.R

```diff
@@ -124,6 +132,30 @@ project <- function(x, R, si, n_sim = 100, n_days = 7,
     stop(msg)
   }
 
+  if (!is.null(time_change)) {
+    if (!is.numeric(time_change)) {
+      msg <- sprintf("`time_change` must be `numeric`, but is a `%s`",
+              paste(class(time_change), collapse = ", "))
+      stop(msg)
+    }
+
+    n_time_periods <- length(time_change) + 1
+
+    if (!is.list(R)) {
```

Comment:
Is it necessary that R is a list? I would suggest to modify this so that if R is a numeric vector, convert it to a list, which appears to happen on line 224

Reply:
Can do; is the use-case you have in mind is a a vector of length `length(time_change) + 1`, with each value a fixed R for the corresponding time period? Just to make sure we're on the same page: so far we consider the input as a distribution of values for each time period. So R needs to be a list, but could be a vector in the particular case where R is fixed within windows.

I'll wait for you to confirm and then will do the requested change.

Reply:
Now done as of a677591d8499847c86831fabe4e80c6b7518a703